### ISSUE TITLE
doc/site:  Components should link to component - specific documentation

### ISSUE THREAD

**Issue by fgheysels** [arcus-azure/arcus#226]
On the Arcus website (https://arcus-azure.net), we have a [section](https://arcus-azure.net/components/) that lists all the components.

When you click on one of those components, you get redirected to the github project page.
Wouldn't it be better if we link to the component-specific documentation for that component instead ?  For instance, if you click on 'Security', you get redirected to security.arcus-azure.net instead of being redirected to https://github.com/arcus-azure/arcus.security

**Comment by stijnmoreels** [comment]
Absolutely! Should indeed be redirected to the docs site of the component. The docs sites also seems like a great start point to learn more about the library instead of the GitHub page. 👍

**Comment by stijnmoreels** [comment]
Not sure though, if we (or I) can already take action on this are we are migrating towards our new general documentation site.

**Comment by fgheysels** [comment]
When is the new general doc site due ?

**Comment by stijnmoreels** [comment]
> When is the new general doc site due ?

<img src="https://media0.giphy.com/media/QCIRnYEiz8HokD5ZcF/giphy.gif"/>
